sale_usability_return_invoicing/wizard/__init__.py

```diff
@@ -1,6 +0,0 @@
-##############################################################################
-# For copyright and license notices, see __manifest__.py file in module root
-# directory
-##############################################################################
-from . import stock_return_picking_line
```

Comment:
I notice that this two wizards were not migrated to sale_stock_ux. are deprecated?

Reply:
becase now it's not necesary odoo incorporate in stock_account module

Reply:
got it